pallets/funding/src/tests/2_evaluation.rs

```diff
@@ -76,6 +76,158 @@ mod round_flow {
 			let project_status = inst.get_project_details(project_id).status;
 			assert_eq!(project_status, ProjectStatus::FundingFailed);
 		}
+
+		#[test]
+		fn different_decimals_ct_still_works_as_expected() {
```

Comment:
Nit: Maybe not call it ct_still_works but rather ct_works. As we know that we have a new decimal system, but after this stack its just the system, so new people coming in might be confused. Like I said small nit.